packages/check-es-compat/bin/cli.mjs

```diff
@@ -46,3 +47,41 @@ async function execute(files) {
 
   return { hasErrors: results.some((result) => result.errorCount > 0) };
 }
+
+function parseArguments(args) {
+    const files = [];
+    const polyfills = [];
+    let nextArgIsPolyfills = false;
+
+    for (const arg of args) {
+        if (nextArgIsPolyfills) {
+            nextArgIsPolyfills = false;
+            polyfills.push(...splitPolyfillsArgument(arg));
+            continue;
+        }
+
+        if (arg.startsWith('--polyfills')) {
+            if (arg.startsWith('--polyfills=')) {
+                polyfills.push(...splitPolyfillsArgument(arg.slice(12)));
+            } else {
+                nextArgIsPolyfills = true;
+            }
+
+            continue;
+        }
+
+        files.push(arg);
+    }
+
+    return { files, polyfills };
+}
+
+function splitPolyfillsArgument(polyfills) {
```

Comment:
This is very specific, and in future (in #90 for ES2023 actually) would need updating if we add similar polyfills.

Another reason to use a file for passing in the polyfills list?